GVRf/Framework/src/org/gearvrf/GVRViewManager.java

```diff
@@ -667,7 +675,9 @@ public void beforeDrawEyes() {
 
             doMemoryManagementAndPerFrameCallbacks();
 
-            mScript.onStep();
+            GVRViewManager.this.getEventManager().sendEvent(
```

Comment:
so, what happens inside this function when there are no scripts?  (i tried skimming it, but am too tired at the moment).  what line number inside sendEvent() does it exit out?